### DISCUSSION TITLE
Original Graph

### DISCUSSION THREAD

**Discussion by frogfrogfrogfrogfrog** [monkeytypegame/monkeytype#677] [Ideas] [answered]
I do like the idea of using error-rate rather than using the amount of correct keys so the accuracy and speed wouldn't overlap or be so close to each other but I still kind of liked the original graph so could you make it toggleable from the original graph to using error-rate.

**Comment by pingshunhuangalex** [comment]
The chart looks quite confusing IMO. Everthing is still accuracy including hover over tooltip, except for the axis title which is the only clue that's telling people the value displayed is something else...

**Comment by frogfrogfrogfrogfrog** [comment]
Yeah I agree. I think you should be able to toggle it from accuracy to error rate where the accuracy graph is just the original graph but the error rate displays the actual error rate instead of accuracy. 

So perhaps Miodec could add a button called "Toggle Accuracy Type" or something like that

**Comment by Miodec** [comment, marked as answer]
I don't know, i kinda wanna limit the amount of very niche settings. I've added error rate to the popup (aswell as keeping accuracy). Do you think that helps?

**Reply by frogfrogfrogfrogfrog** [reply]
Yeah it definitely helps that you've added proper error rate. The problem is that it isn't rounding correctly.

![image](https://user-images.githubusercontent.com/58377021/101655985-72c6b180-39ff-11eb-9c2f-b06df71d0da2.png)

**Reply by Miodec** [reply]
Whoops

**Reply by frogfrogfrogfrogfrog** [reply]
Also, while that helps without a doubt. And yes I can just use different time zones like from now to a week ago I still kind of dislike the fact that my graph looks like this

![Screenshot 2020-12-10 at 11 58 51 AM](https://user-images.githubusercontent.com/58377021/101818404-0a500100-3ae1-11eb-90e3-dd74e3f35627.png)
